src/auto-complete.directive.ts

```diff
@@ -355,6 +341,16 @@ export class NguiAutoCompleteDirective implements OnInit, OnChanges, AfterViewIn
         });
     }
 
+    public selectCustomValue2 = (text: string) => {
```

Comment:
@ctrahhuk This method seems to be poorly named. Wouldn't it make more sense to call it something like selectFilter?

Reply:
u're right, missed renaming this method :)